src/middlewared/middlewared/migration/0005_tunables_linux.py

```diff
@@ -0,0 +1,7 @@
+async def migrate(middleware):
```

Comment:
Why not a regular migration for this?

Reply:
That will be executed for users in CORE as well - however when they upgrade to SCALE, it won't be run again whereas this will be